lib/streamlit/web/server/server_util.py

```diff
@@ -121,16 +121,8 @@ def emit_endpoint_deprecation_notice(
     handler: tornado.web.RequestHandler, new_path: str
 ) -> None:
     """
-    Emits the warning about deprecation of HTTP endpoint both in the HTTP header
-    and in the application logs.
+    Emits the warning about deprecation of HTTP endpoint in the HTTP header.
     """
-    # Use repr function to avoid log injection attack.
-    # See: https://codeql.github.com/codeql-query-help/python/py-log-injection/
-    _LOGGER.warning(
```

Comment:
Looks like CodeQL is correct here, and  _LOGGER becomes unused

Reply:
Right. Fixed.